app/views/searches/advanced.html.erb

```diff
@@ -0,0 +1,28 @@
+<% @title = t("advanced_search") %>
```

Comment:
Setting `@title` in view seems to be a [convention followed in all of the templates](https://github.com/rubygems/rubygems.org/blob/master/app/views/rubygems/show.html.erb#L1).